src/main/java/io/strimzi/kafka/bridge/http/converter/HttpTextMessageConverter.java

```diff
@@ -43,14 +44,14 @@ public ProducerRecord<byte[], byte[]> toKafkaRecord(String kafkaTopic, Integer p
                 if (!keyNode.isTextual()) {
                     throw new IllegalStateException("Because the embedded format is 'text', the key must be a string");
                 }
-                key = keyNode.asText().getBytes();
+                key = keyNode.asText().getBytes(Charset.forName("UTF-8"));
```

Comment:
Same as above, are these changes really related to the metrics? How?

Reply:
No. This is due to some validation/spotbug error I got after rebasing.

Reply:
Again, why it's not happening today in the main?